src/main/java/frc/robot/systems/BlueMech.java

```diff
@@ -3,39 +3,40 @@
 // WPILib Imports
 
 // Third party Hardware Imports
-import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj.Solenoid;
+import edu.wpi.first.wpilibj.Timer;
 
 // Robot Imports
 import frc.robot.TeleopInput;
 import frc.robot.HardwareMap;
 
-public class FSMSystem {
+public class BlueMech {
 	/* ======================== Constants ======================== */
 	// FSM state definitions
 	public enum FSMState {
-		START_STATE,
-		OTHER_STATE
+		IDLE,
+		FIRING
 	}
 
-	private static final float MOTOR_RUN_POWER = 0.1f;
+	private static final int PUSH_TIME = 3;
 
 	/* ======================== Private variables ======================== */
 	private FSMState currentState;
 
-	// Hardware devices should be owned by one and only one system. They must
-	// be private to their owner system and may not be used elsewhere.
-	private CANSparkMax exampleMotor;
+	private Solenoid push;
```

Comment:
nit: variable name can be more descriptive (maybe pushSolenoid?) for ease of code readability

Reply:
Renamed to pushSolenoid